CMPUT_301_Project/app/src/main/java/com/example/cmput_301_project/EventHabitAdapter.java

```diff
@@ -0,0 +1,142 @@
+package com.example.cmput_301_project;
+
+import android.app.Activity;
+import android.media.metrics.Event;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AlertDialog;
+import androidx.fragment.app.FragmentActivity;
+import androidx.fragment.app.FragmentManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import java.text.DateFormat;
+import java.util.List;
+
+public class EventHabitAdapter extends RecyclerView.Adapter<EventHabitAdapter.ItemVH>{
+    private static final String TAG="Adapter";
+    //Account userAccount = AccountData.create().getActiveUserAccount();
```

Comment:
Normally I'd ask to remove commented code, but we will need this for connectivity later (no action needed)